### DISCUSSION TITLE
Collabora container cannot open Documents on Nextcloud

### DISCUSSION THREAD

**Discussion by progecon** [nextcloud/all-in-one#4200] [Questions] [answered]
### Steps to reproduce
1. Make sure to follow https://github.com/nextcloud/all-in-one/discussions/1358
2. Deploy Nextcloud AIO using Docker.
3. Attempt to integrate Collabora Online server within the Nextcloud AIO instance.
4. Encounter errors related to coolwsd process initialization, specifically with mounting/unmounting directories.

### Expected behavior 
Collabora Online server should successfully initialize and integrate with Nextcloud AIO, allowing for document editing and collaboration within the Nextcloud interface.

### Actual behavior 
The coolwsd process encounters an "Operation not permitted" error during initialization, specifically when executing the /usr/bin/coolmount command, indicating a potential permissions, binary absence, or misconfiguration issue.

### Host OS 
Docker on Ubuntu 22.04

#### Nextcloud AIO version 
Nextcloud AIO v7.11.2

#### Current channel 
latest

##### 1. Collabora Container logs
[collabora-container-log.txt](https://github.com/nextcloud/all-in-one/files/14221865/collabora-container-log.txt)

##### 2. Output of the main nextcloud container
[nextcloud-container-output.txt](https://github.com/nextcloud/all-in-one/files/14221864/nextcloud-container-output.txt)

**Comment by szaimen** [comment, marked as answer]
Hi, can you follow https://github.com/nextcloud/all-in-one/discussions/1358?

**Reply by progecon** [reply]
Hey @szaimen , thank you for digging into this. I already ran all the mentioned steps before, but did so again now to be on the safe side. None of these steps worked, I still get the following error when trying to open the sample docx-file in nextcloud: `Failed to establish socket connection or socket connection closed unexpectedly. The reverse proxy might be misconfigured, please contact the adminsitrator.`
[collabora-log_again.txt](https://github.com/nextcloud/all-in-one/files/14233335/collabora-log_again.txt)
[hosting.discovery_again.txt](https://github.com/nextcloud/all-in-one/files/14233336/hosting.discovery_again.txt)

**Reply by szaimen** [reply]
Possibly this is your problem?
![image](https://github.com/nextcloud/all-in-one/assets/42591237/65a5ac11-3607-44dc-b196-cee19c824782)

Did you adjust the wopi-allowlist as advised in the thread I linked?

**Reply by progecon** [reply]
Sorry for my late answer. Yes, I did add ,0.0.0.0/0 to it as requested before opening an issue here. I did so again now, and did not get a different result. When trying to open Welcome to Nextcloud Hub.docx, I get the attached error message. Leaving the wopi-allowlist completely empty leaves the same result.

Just to make it clear, I am not using cloudflare.

**Comment by progecon** [comment]
Sorry for my late answer. Yes, I did add ,0.0.0.0/0 to it as requested before opening an issue here. I did so again now, and did not get a different result. When trying to open Welcome to Nextcloud Hub.docx, I get the attached error message. Leaving the wopi-allowlist completely empty leaves the same result.

Just to make it clear, I am not using cloudflare.

![grafik](https://github.com/nextcloud/all-in-one/assets/109902876/53dac1be-0303-4855-844c-a7e5ffd4ffcf)

**Reply by Aurial** [reply]
Out of interest, have you tried putting your external IP of your server into the WOPI allow list?  I've had problems in the past and adding my external IP to the list has solved them.

**Reply by progecon** [reply]
Yes I did. In fact, my external server IP is the first entry in my WOPI allow list.
